librosa/core/constantq.py

```diff
@@ -627,20 +625,22 @@ def icqt(
     # Get the top octave of frequencies
     n_bins = C.shape[-2]
 
+    n_octaves = int(np.ceil(float(n_bins) / bins_per_octave))
```

Comment:
not sure why you had to move up this definition of `n_octaves`, but OK

Reply:
Mostly just for legibility